.github/workflows/publish-command.yml

```diff
@@ -94,15 +94,39 @@ jobs:
           ci_credentials ${{ github.event.inputs.connector }}
         env:
           GCP_GSM_CREDENTIALS: ${{ secrets.GCP_GSM_CREDENTIALS }}
-      - run: |
+      - name: Prepare Sentry Release
+        if: startsWith(github.event.inputs.connector, 'connectors')
+        run: |
+          curl -sL https://sentry.io/get-cli/ | bash
```

Comment:
does it make sense to commit this file to git? Then if e.g. sentry.io goes down, our ability to release isn't impacted